README.md

```diff
@@ -67,6 +67,29 @@ public async Task Example()
 }
 ```
 
+### DI Usage
+
+```csharp
+// Register your feature flag provider
+builder.Services.AddOpenFeature(static builder =>
+{
+    builder.Services.TryAddEnumerable(ServiceDescriptor.Singleton<FeatureProvider, SomeFeatureProvider>());
+    builder.TryAddOpenFeatureClient(SomeFeatureProvider.Name);
```

Comment:
Is it possible to do request scoped DI? It would be really great to include an example where you sent request specific context in the client. Since it's a slightly more advanced topic, perhaps it could be included in the evaluation context section later in the doc.

Reply:
It looks like it should work. Here's an example from .NET FeatureManagement.

https://github.com/microsoft/FeatureManagement-Dotnet?tab=readme-ov-file#using-httpcontext